sdk/kubernetesconfiguration/Microsoft.Azure.Management.KubernetesConfiguration/src/Microsoft.Azure.Management.KubernetesConfiguration.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.KubernetesConfiguration</PackageId>
+    <Description>Provides Microsoft Azure KubernetesConfiguration management operations including the ability to create, update and delete runbooks and schedules.</Description>
+    <Version>3.8.2-preview</Version>
+    <!-- Remove the Preview description in PackageReleaseNotes once the preview is removed.-->
+    <AssemblyTitle>Microsoft Azure KubernetesConfiguration Management Library</AssemblyTitle>
+    <AssemblyName>Microsoft.Azure.Management.KubernetesConfiguration</AssemblyName>
+    <PackageTags>KubernetesConfiguration;</PackageTags>
+    <PackageReleaseNotes><![CDATA[
+New features:
+   Adding support for minute schedules
+      ]]>
+	  </PackageReleaseNotes>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
+  </PropertyGroup>
+
+  <!-- Please do not move/edit code below this line -->
+  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.RP.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
+  <ItemGroup>
+    <PackageReference Update="Microsoft.Rest.ClientRuntime" Version="2.3.21" />
+    <PackageReference Update="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Update="System.Net.Http" Version="4.3.4" />
+  </ItemGroup>
```

Comment:
Please remove these references, as they are already defined in `AzSdk.reference.props`

Reply:
Removed